docs/authentication/saml/okta.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Okta Workforce
+description: Learn how to integrate Okta Workforce with Clerk using SAML SSO.
+---
+
+# Okta Workforce
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+    },
+    {
+      title: "Add the appropriate Clerk SDK to your application",
+      link: "/docs/quickstarts/overview",
+    },
+  ]}
+>
+
+- Use Okta Workforce to enable single sign-on (SSO) via SAML for your Clerk application.
+
+</TutorialHero>
+
+Integrating Okta Workforce with Clerk allows you to use it as an Identity Provider (IdP) for your Clerk application to authenticate your users and manage their access to your application via SAML SSO.
+
+## Tutorial
+
+<Steps>
+
+### Set up Enterprise Connection in Clerk
+
+To create a SAML connection in Clerk:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com).
+2. In the navigation sidebar, select **User & Authentication > [Enterprise Connections](https://dashboard.clerk.com/last-active?path=user-authentication/enterprise-connections)**.
+3. Select the **Create connection** button.
+4. You will be presented with a modal to create a new connection. Fill in the required fields.
+- **Name** - A unique name for the connection. This is so you can tell your connections apart.
+- **Domain** - The email domain for which you will enable Enterprise SSO.
+- **Identity Provider** - The IdP you are using. For this tutorial, select **Okta Workforce**.
+5. Once you have filled in these fields, you will be directed to the **Connection** page which has the configuration settings for the connection you just created. **Do not select the Enable connection toggle just yet.** You need to fully configure your IdP first before exposing your connection to your users while you're in the, sometimes long, process of configuring and testing it.
+6. Leave this page open. You will need to come back to it to complete the setup.
+
+### Create a new Enterprise Application in Okta
+
+To create a new Enterprise Application in Okta:
+
+1. Navigate to [Okta](https://www.okta.com/) and sign in.
+2. In the Okta dashboard, select **Admin** in the top right corner.
+3. In the navigation sidebar,  select the **Applications** dropdown and select **Applications**.
+4. Select the **Create App Integration** button.
+5. In the **Create a new app integration** modal, select the **SAML 2.0** option and select the **Next** button.
+6. Once redirected to the **Create SAML Integration** page, fill in the **General Settings** fields. An **App name** is required.  You can also upload an **App logo** if you wish.
```

Comment:
```suggestion
6. Once redirected to the **Create SAML Integration** page, fill in the **General Settings** fields. An **App name** is required.
```
Not a big deal, but might as well cut any instructions that aren't strictly necessary